src/common/common-types.ts

```diff
@@ -228,8 +228,8 @@ export interface PythonInfo {
 }
 
 export interface FfmpegInfo {
-    readonly ffmpeg: string;
-    readonly ffprobe: string;
+    readonly ffmpeg?: string;
+    readonly ffprobe?: string;
 }
```

Comment:
I want to slightly change the type. In the code, it's actually either (1) we know both or (2) we know neither. That's a lot stronger than "either might or might not be known".

```ts
export type FfmpegInfo = {
    readonly ffmpeg: string;
    readonly ffprobe: string;
} | {
    readonly ffmpeg: undefined;
    readonly ffprobe: undefined;
}
```